src/main/java/io/github/cottonmc/cotton/gui/widget/WScrollBar.java

```diff
@@ -18,7 +18,7 @@
 import static io.github.cottonmc.cotton.gui.client.BackgroundPainter.createNinePatch;
 
 public class WScrollBar extends WWidget {
-	private static final int SCROLLING_SPEED = 4;
+	private int scrollingSpeed = 4;
```

Comment:
The default value should be refactored into its own `public static final` constant field. The `get/setScrollingSpeed` docs can then use `{@value #DEFAULT_SCROLLING_SPEED}` instead of duplicating the constant value three times.